File: Easy/damage.py
# https://edabit.com/challenge/HSHHkdRYXfgfZSqri
def damage(damage, speed, time):
    # timers= ["second","minute",'hour']
    timer = 0
    if speed < 0 or damage <0:
        return 'invalid'
    else:
        if time == "second":
            timer=1
        elif time== "minute":
            timer = 60
        else:
            timer= 3600
    return damage*(speed*timer)

print(damage(40, 5, "second"), 200)
print(damage(100, 1, "minute"), 6000)
print(damage(2, 100, "hour"), 720000)

File: javascript/Very_Easy/damage.js
// https://edabit.com/challenge/g7FhPFYdZACYeJTSk


function damage(damage, speed, time) {
	if(damage<0||speed<0){
        return 'invalid'
    }
    const mutiplyer = time==='second'?1:time==='minute'?60:3600;
    return damage*speed*mutiplyer
}


console.log(damage(40, 5, "second"), 200)
console.log(damage(100, 1, "minute"), 6000)
console.log(damage(2, 100, "hour"), 720000)
console.log(damage(20, 0.5, "minute"), 600)
console.log(damage(2, 400, "hour"), 2880000)
console.log(damage(-23, 20, "second"), "invalid")
console.log(damage(-23, -5, "second"), "invalid")
console.log(damage(20, -492321, "hour"), "invalid")